GlucoseOnFhir/DeviceViewController.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  DeviceViewController.swift
+//  GlucoseOnFhir
+//
+//  Created by Kevin Tallevi on 1/16/17.
+//  Copyright © 2017 eHealth Innovation. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SMART
+
+class DeviceViewController: UITableViewController {
+    public var device: Device!
+    let cellIdentifier = "DeviceCellIdentifier"
+    let sectionHeaderHeight: CGFloat = 75
+    
+    enum Section : Int {
+        case section0, section1, section2, section3, count
+        
+        public func description() -> String {
+            switch self {
+            case .section0:
+                return "Identifier"
+            case .section1:
+                return "Type"
+            case .section2:
+                return "Manufacturer"
+            case .section3:
+                return "Model"
+            case .count:
+                fatalError("invalid")
+            }
+        }
+        
+        public func rowCount() -> Int {
+            switch self {
+            case .section0:
+                return 4
+            case .section1:
+                return 4
+            case .section2:
+                return 1
+            case .section3:
+                return 1
+            case .count:
+                fatalError("invalid")
+            }
+        }
+        
+        enum Section0 : Int {
+            case typeCodingSystem, typeCodingCode, system, value, count
+        }
+        enum Section1 : Int {
+            case codingSystem, codingCode, codingDisplay, text, count
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    // MARK: Table data source methods
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case Section.section0.rawValue:
+            return Section.section0.rowCount()
+        case Section.section1.rawValue:
+            return Section.section1.rowCount()
+        case Section.section2.rawValue:
+            return Section.section2.rowCount()
+        case Section.section3.rawValue:
+            return Section.section3.rowCount()
+        default:
+            return 0
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath as IndexPath) as UITableViewCell
+        
+        switch indexPath.section {
+        case Section.section0.rawValue:
+            switch(indexPath.row) {
```

Comment:
Again, create enums from the ints and it cleans this up a lot. (Plus the compiler warns you if you are missing a case.)